subprojects/core/src/integTest/groovy/org/gradle/api/MutationFailureFromConfigureEachDomainObjectContainerIntegrationTest.groovy

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2021 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.api
+
+import spock.lang.Unroll
+
+abstract class MutationFailureFromConfigureEachDomainObjectContainerIntegrationTest extends AbstractDomainObjectContainerIntegrationTest {
```

Comment:
Given that all these classes now only have one test method, maybe it would make sense to put them all in one file. I wonder how we could make these things easier to split up.